.github/commitlint.config.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
     "header-max-length": [0],
     // Disable some common mistyped scopes and some that should be used
     "scope-enum": [2, "never", [
-      "kumacp", "kumadp", "kumacni", "kumainit", "*", "madr", "test", "ci", "perf"
-    ]]
+      "kumacp", "kumadp", "kumacni", "kumainit", "*", "madr", "test", "ci", "perf", "policies"
```

Comment:
why madr is banned? what should I use instead of
```
docs(madr): offline signing tokens
```

Reply:
MADR (I've disallowed one to avoid having both)